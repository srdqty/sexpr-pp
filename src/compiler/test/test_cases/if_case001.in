(if #t 3 4)
(if #f 5 6)
(if 1 2 3)
