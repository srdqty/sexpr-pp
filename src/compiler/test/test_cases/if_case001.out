(If (Constant #t) (Constant 3) (Constant 4))
(If (Constant #f) (Constant 5) (Constant 6))
(If (Constant 1) (Constant 2) (Constant 3))
